src/test/java/com/telefonica/iot/cygnus/sinks/OrionHDFSSinkTest.java

```diff
@@ -466,6 +467,7 @@ private Context createContext(boolean useDeprecatedParams, String fileFormat) {
             context.put("hdfs_username", hdfsUsername);
         } // if else
         
+        context.put("default_csv_separator",defaultCsvSeparator);
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/93781aa1f586022ef98d1f1612a7c737add79f30